scripts/generate_ssl_debug_helpers.py

```diff
@@ -328,7 +328,7 @@ def __str__(self):
             {translation_table}
                 }};
 
-                return "UNKOWN";
+                return "UNKNOWN";
```

Comment:
This was previously fixed in #5732 at https://github.com/Mbed-TLS/mbedtls/pull/5732/files#diff-c1d4f10f0fef89231d91756c0b67ee38bcbed022a4b80f0ea3b44fb4c0e4b845R292, so how come it's wrong in-tree?

Reply:
47854e638b48de67d7daa195b9ebc2f0cbd1eec6 reverted `return "UNKOWN_VAULE";` to the  return with the oopsi.

Reply:
Ah yes, but for "UNKOWN" we only fixed one of two instances in the file back in #5732 - there is so much duplicated template around this line that it wasn't obvious to me yesterday evening